test/studies/hpcc/HPL/vass/hpl.hpcc2012.notest

```diff
@@ -0,0 +1,8 @@
+This test fails to work with the new "deallocate arrays at the end of
```

Comment:
If so, I didn't realize.  If it is, I think we should stop doing so rather than hold off this improvement over it.  Let's discuss further in this week's perf meeting.